CenterForOpenScience/cos.io
# [PLAT-823] Set up rss feed for cos.io

## Purpose 

Our blog needs an RSS feed, Wagtail makes that easy, let's do it!

## Changes

- Adds BlogFeed class that automatically tranforms blogs into RSS.

## QA Notes 

Visit staging-cos.io/blog/feed/ and see that the RSS template has all the blog posts with accurate titles and descriptions.

## Tickets
https://openscience.atlassian.net/browse/PLAT-823
